plugins/experimental/stale_response/ts_wrap.h

```diff
@@ -0,0 +1,42 @@
+/** @file
+
+  Wrapper for ts.h, to allow for compatibility with multiple major versions.
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#pragma once
+
+#include "ts/ts.h"
+
+#if TS_VERSION_MAJOR >= 10
```

Comment:
This will always be true.

Reply:
My intention was to leave this version conditional incase someone wants to backport it, either in ASF or internal. We in Yahoo, for instance, need it internally for 9.x. Does that make sense? I'm convincible if you'd still like me to remove this.

My comment here applies to comparison of major version to 10 and above. Your other comments about older versions I'll definitely take out. Those were here because this is an ASF port of an old internal plugin.